pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/kube_ingress_reconciler.go

```diff
@@ -190,7 +190,8 @@ func createRawIngress(scheme *runtime.Scheme, isvc *v1beta1api.InferenceService,
 			Annotations: isvc.Annotations,
 		},
 		Spec: netv1.IngressSpec{
-			Rules: rules,
+			IngressClassName: ingressConfig.IngressClassName,
```

Comment:
If it's `nil` all IngressClassName will be empty and will use default ingress in the cluster (https://kubernetes.io/docs/concepts/services-networking/ingress/#default-ingress-class) . In GKE case it will use create Google Ingress. 
If inference service owner want to use specific ingress then they'll have to add ingress class annotation (kubernetes.io/ingress.class)